_sass/blocks/explore/_map.scss

```diff
@@ -12,23 +12,32 @@ eiti-slider {
 
 .region-map {
   display: block;
-  height: 250px;
   margin-bottom: 1em;
-  max-height: 30vh; // 30% of the viewport height
+  max-height: 50vh; // 50% of the viewport height
   width: 100%;
 }
 
 .filter-description {
+  @include heading(para-xl);
+  font-weight: $weight-bold;
+  letter-spacing: -0.5px;
+
   .filter-part {
     border-bottom: 2px dotted $neutral-gray;
     color: inherit;
     text-decoration: none;
   }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 31 to appear before line 25. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets